src/UI/pages/source_page.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect } from 'react';
+import {Paper, Typography, Box, Button, Container, Grid } from '@mui/material';
+import { getSourceInfo } from '../state/sourceSlice';
+import store, { AppDispatch } from '../state/store';
+import { useAppSelector } from '../state/hooks';
+import { useDispatch } from 'react-redux';
+import SingleSource from '../components/sources/singleSource';
+
+
+
+function SourcesView(): JSX.Element{
+    const source_info = useAppSelector((state) => state.source.source_info)
+    const source_info_status = useAppSelector((state) => state.source.source_info_status)
+
+    console.log(source_info);
```

Comment:
Console here

Reply:
Thanks so much. Will do that

Reply:
No worries. If I can get my git to behave I'll have a look at it locally too - but I'm getting an invalid path which is fun